File: sum-of-numbers-from-0-to-n/sum-of-numbers-from-0-to-n.js
var SequenceSum = (function() {
  function SequenceSum() {}

  SequenceSum.showSequence = function(count) {
  if(count==0){return "0=0";}
  else if(count<0){return count+"<0";} 
var n = new Array(count).fill(0);
n = n.map(function(a,b){return a+b+1})
return "0+"+n.join("+") + " = "+ n.reduce(function(a,b){return a+b})
  };
  return SequenceSum;
})();

File: sum-of-numbers-from-0-to-n/sum-of-numbers-from-0-to-n.py
def show_sequence(n):
    if n==0:
        return '0=0'
    if n<0:
        return str(n)+"<0"
    return "+".join([str(i) for i in range(n+1)]) +" = " + str(sum([i for i in range(n+1)]))